tests/gateways/test_repodata_gateway.py

```diff
@@ -19,11 +24,233 @@
     ProxyError,
     UnavailableInvalidChannel,
 )
+from conda.gateways.connection import HTTPError, InvalidSchema, RequestsProxyError, SSLError
```

Comment:
Let's move the lock related tests into a new module `tests/gateways/test_lock.py`